data/json/recipes/recipe_food.json

```diff
@@ -5321,6 +5321,49 @@
     "tools": [ [ [ "surface_heat", 2, "LIST" ] ] ],
     "components": [ [ [ "sweet_fruit", 1, "LIST" ], [ "coconut", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
```

Comment:
This is probably not right (same goes for the other ones).

Reply:
Should it be `"NO_EXERCISE"`? Since you are literally putting fruit in a jar then filling the jar with alcohol and calling it done.

Reply:
Probably.